torch/fx/experimental/proxy_tensor.py

```diff
@@ -1594,14 +1594,14 @@ def __init__(self, scope_root: GraphModule) -> None:
         tracer = self
 
         class AttrProxy(_AttrProxy):
-            def __init__(self, base: Module, path: str) -> None:
+            def __init__(self, base: Module, path: str, **kwargs) -> None:
```

Comment:
```suggestion
            def __init__(self, base: Module, path: str, **kwargs: object) -> None:
```
I don't think there's any way to convince the type annotations to be more specific. Ideally we'd really want something like:
```
_TModule = TypeVar("_TModule", bound=Module)

def __init__(self, base: _TModule, path: str, **kwargs: _TModule.__init_subclass__.kwargs) -> None:
```
but that's not really a valid thing.